artifactory/services/users.go

```diff
@@ -157,3 +157,30 @@ func (us *UserService) DeleteUser(name string) error {
 	}
 	return errorutils.CheckResponseStatusWithBody(resp, body, http.StatusOK)
 }
+
+func (us *UserService) GetLockedUsers() ([]*string, error) {
```

Comment:
Is there a reason for the array of pointers?
```suggestion
func (us *UserService) GetLockedUsers() ([]string, error) {
```